.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  # Enables manually triggers
+  workflow_dispatch:
+  push:
+    branches:
+      - "radix-prod-env"
+    tags:
+      - "v**"
```

Comment:
should also update the "master-build-and-deploy" to remove tags trigger.
No need to run the CI needlessly.